primitives/core/Cargo.toml

```diff
@@ -66,7 +66,7 @@ hex = { version = "0.4", default-features = false, optional = true }
 twox-hash = { version = "1.5.0", default-features = false, optional = true }
 libsecp256k1 = { version = "0.6", default-features = false, features = ["hmac", "static-context"], optional = true }
 merlin = { version = "2.0", default-features = false, optional = true }
-
+ss58-registry = { git = "https://github.com/paritytech/ss58-registry.git", branch = "main" }
```

Comment:
I want that we use it from crates.io please.

Reply:
It's coming: https://github.com/paritytech/ss58-registry/pull/14